business-registry-dashboard/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "business-registry-dashboard",
   "private": true,
   "type": "module",
-  "version": "0.0.20",
+  "version": "0.0.22",
```

Comment:
I'm assuming Karim's PR will be merged before this one. I'll have to rebase anyway probably.

Reply:
I've updated the app version to 0.0.21 as, surprisingly, my PR is ready before Karim's :)